core/io/file_access_compressed.cpp

```diff
@@ -99,7 +99,7 @@ Error FileAccessCompressed::_open(const String &p_path, int p_mode_flags) {
 	ERR_FAIL_COND_V(p_mode_flags == READ_WRITE, ERR_UNAVAILABLE);
 
 	if (f.is_valid()) {
```

Comment:
Last consistency nitpick, it seems like all `_close` implementations have early returns for the case where there's nothing to close, so the `is_valid()` tests could be removed here and in the destructor. Same for `FAEncrypted` and `FANetwork`.

Reply:
Done.